tests/conftest.py

```diff
@@ -604,6 +604,8 @@ def collect_techsupport_on_dut(request, a_dut):
         res = a_dut.shell("generate_dump -s \"-2 hours\"")
         fname = res['stdout_lines'][-1]
         a_dut.fetch(src=fname, dest="logs/{}".format(testname))
+        res = a_dut.shell("sudo tar czf tmp.tgz /tmp")
```

Comment:
What's the purpose for tar and fetch ```tmp``` from DUT?

Reply:
removed, not needed any more.